### ISSUE TITLE
ng-change

### ISSUE THREAD

**Issue by rmpt** [Alberplz/angular-colorpicker-directive#10]
ng-change not working. It would be nice to listen to onchange event so we know on our scope that something has changed.

**Comment by cmckni3** [comment]
Try https://github.com/zefoy/ngx-color-picker. This project seems unmaintained

**Comment by tmescic** [comment]
+1

**Comment by bayucandra** [comment]
@cmckni3 Just check your link, seems it is for Angular not for AngularJS.

**Comment by cmckni3** [comment]
Yeah @bayucandra I posted on the wrong project.

I thought I was under the [Angular project](https://github.com/Alberplz/angular2-color-picker)

**Comment by cmckni3** [comment]
Anyways. Still seems as though both projects by @Alberplz are unmaintained.

**Comment by bayucandra** [comment]
Yes, that is right. Personally, I feel this project to good to be left. Even I know AngularJS will end the cycle soon :).

**Comment by CharlyPoppins** [comment]
Hi, still using it on a legacy project.

My way to detect if the color has changed :
```
this.$scope.$on('color-changed', () => {
    console.log('color-changed', this.colorModel)
})```
